extra/html-extra/HtmlExtension.php

```diff
@@ -110,4 +111,27 @@ public static function htmlClasses(...$args): string
 
         return implode(' ', array_unique($classes));
     }
+
+    /**
+     * @param array{
+     *     base: string|list<string|null>|null,
+     *     variants: array<string, array<string, string|array<string>>>,
+     *     compoundVariants: array<array<string, string|array<string>>>,
+     *     defaultVariants: array<string, string>
+     *  } $cva
+     *
+     * base some base class you want to have in every matching recipes
+     * variants your recipes class
+     * compoundVariants compounds allow you to add extra class when multiple variation are matching in the same time
+     * defaultVariants allow you to add a default class when no recipe is matching
+     */
+    public function cva(array $cva): CVA
```

Comment:
Why not use the same signature as the CVA constructor?

Reply:
It has already been discussed on the SymfonyUX repo, but the main reason is to keep align with the js implementation.